### ISSUE TITLE
Venue Search Results

### ISSUE THREAD

**Issue by ctrembath** [club-soda/club-soda-guide#15]
As a customer who has searched for venues,
I'd like to see relevant venue cards in a list view with their primary details,
so I can make an informed decision whether to visit the venue.

**Comment by iteles** [comment]
@jussi-club-soda What would you like the default ordering to be here seeing as we don't have CS-determines weightings for venues like we do for the drinks?
Should we order them by Club Soda score?

**Comment by jussi-club-soda** [comment]
yep, in the absence of geolocation, Club Soda Score sounds like a good way or sorting! though hopefully there will be lots with a maximum score, so does it need a secondary sorting variable too?

**Comment by iteles** [comment]
@jussi-club-soda Good shout, even if we ultimately want to sort them as geolocated, do we need a secondary sorting variable for when someone has filtered to a specific CS Score? Alphabetical perhaps?

**Comment by iteles** [comment]
@jussi-club-soda 👍 
@SimonLab Jussi's correct, they should be by CS score _first_ and only then alphabetically _within_ those venues that have the _same score_.

For example, if we only had the 4 venues below, we would want the following order:
+ Acqua7 (3 score, but 'c' comes before the 'd' in Adam)
+ Adam & Eve
+ 4 degrees
+ Abbey
![image](https://user-images.githubusercontent.com/4185328/49604208-4184f800-f985-11e8-8ca8-350512e2d82b.png)
